505.xml

```diff
@@ -0,0 +1,265 @@
+				<Mandatory>Optional</Mandatory>
+				<Type>String</Type>
+				<RangeEnumeration>0..255</RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[Name of the Profile Package.]]></Description>
+			</Item>
+			<Item ID="9">
+				<Name>Profile Package Version</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>String</Type>
+				<RangeEnumeration>0..255</RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[Version of the Profile package.]]></Description>
+			</Item>
+			<Item ID="10">
+				<Name>Profile Update Protocol Support</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Multiple</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration>0..3</RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[This resource indicates what protocols the LwM2M Client implements to retrieve Profiles. The LwM2M server uses this information to decide what URI to include in the Package URI. A LwM2M Server MUST NOT include a URI in the Package URI object that uses a protocol that is unsupported by the LwM2M client. For example, if a LwM2M client indicates that it supports CoAP and CoAPS then a LwM2M Server must not provide an HTTP URI in the Packet URI. The following values are defined by this version of the specification: 
+0: CoAP (as defined in RFC 7252) with the additional support for block-wise transfer. CoAP is the default setting. 
+1 - CoAPS (as defined in RFC 7252) with the additional support for block-wise transfer 
+2 - HTTP 1.1 (as defined in RFC 7230) 
+3 - HTTPS 1.1 (as defined in RFC 7230) Additional values MAY be defined in the future. Any value not understood by the LwM2M Server MUST be ignored.]]></Description>
+			</Item>
+			<Item ID="11">
+				<Name>Profile Update Delivery Method</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Mandatory</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration>0..2</RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[The LwM2M Client uses this resource to
+indicate its support for transferring Profile
+images to the client either via the Package
+Resource (=push) or via the Package URI
+Resource (=pull) mechanism.
+0: Pull only
+1 - Push only
+2 - Both. In this case the LwM2M Server
+MAY choose the preferred mechanism for
+conveying the profile image to the
+LwM2M Client.]]></Description>
+			</Item>
+			<Item ID="12">
+				<Name>Free Memory on SIM</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units>KiB</Units>
+				<Description><![CDATA[Estimated current available amount of storage space on SIM which can store data and software in the LwM2M Device (expressed in kilobytes).]]></Description>
```

Comment:
Do we mean here, and in the next resource, **kilo**bytes (as in 1000 * bytes) or **kibi**bytes (i.e., 1024 * bytes). I assume the latter since that's more common with storage (and what Units field currently says).